NO_PERSON_ERROR = Person with the name ''{0}'' was not found
